unit_tests/system/logger_test.cpp

```diff
@@ -50,12 +54,31 @@ void logger_messages() {
 	LoggerOptions::get_instance().reset();
 }
 
+void set_log_file(std::string log_file_base) {
```

Comment:
Avoid code duplicity.
Keep function for log file name in logger and only keep FilePath staff out of it.